.editorconfig

```diff
@@ -32,7 +32,7 @@ dotnet_sort_system_directives_first                        = true
 # Code-block preferences
 csharp_prefer_braces = true
 csharp_prefer_simple_using_statement                       = true
-csharp_style_namespace_declarations                        = file_scoped:suggestion
+csharp_style_namespace_declarations                        = file_scoped:error
```

Comment:
Locally, everything should be at most a warning, so you can ignore them while you're prototyping. Then, due to `TreatWarningsAsErrors`, the CI build will still fail.

Reply:
I am not sure that dotnet format uses TreatWarningsAsErrors which is why this was needed. Ill try it with warning

Reply:
That's for `dotnet build`. `dotnet format` isn't needed.